frontend/src/validators/expense.js

```diff
@@ -28,7 +29,7 @@ const schema = Joi.object().keys({
     .label('IncurredAt'),
   category: Joi.string().required()
     .label('Category'),
-  payoutMethod: Joi.string().valid(pluck(payoutMethods('en'), 'value'))
+  payoutMethod: Joi.string().valid(pluck(payoutMethods(i18n('en')), 'value')) // i18n.getString not used here
```

Comment:
this sounds very hacky. Why having i18n in a validation?
The value of an attribute should be independent of the locale.